src/char/char.c

```diff
@@ -1587,17 +1587,27 @@ int char_make_new_char_sql(struct char_session_data *sd, const char *name_, int
 	if( sd->found_char[slot] != -1 )
 		return -2; /* character account limit exceeded */
 
+	switch (sex) {
+		case SEX_FEMALE:
+			sex = 'F';
+			break;
+		case SEX_MALE:
+			sex = 'M';
+			break;
+		default:
+			return -2;	// Char Creation Denied
```

Comment:
Since this switch statement is called for all packet versions, wouldn't it fail for those that don't support the gender byte?
As the "default value" for ` sex` for packet versions < 20151001 is `'U'`.